src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -19,8 +19,11 @@
 #include <legacy/ngraph_ops/power.hpp>
 #include <legacy/ngraph_ops/relu_ie.hpp>
 
+#include "gna_plugin_config.hpp"
 #include "backend/gna_limitations.hpp"
+#include "layers/gna_convolution_layer.hpp"
 #include "layers/gna_permute.hpp"
+#include <transformations/utils/transformation_helper.hpp>
```

Comment:
It would be better to use <> for system headers only and "" for all our headers.